packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -557,6 +557,25 @@ describe('ReactDOMSelect', () => {
     }
   });
 
+  it('should warn if selected is set on <option>', () => {
+    spyOnDev(console, 'error');
+
+    ReactTestUtils.renderIntoDocument(
+      <select>
+        <option selected={true} />
+        <option selected={true} />
+      </select>,
+    );
+    if (__DEV__) {
+      expect(console.error.calls.argsFor(0)[0]).toContain(
+        'Use the `defaultValue` or `value` props on <select> instead of ' +
+          'setting `selected` on <option>.',
+      );
+      // Test deduplication
+      expect(console.error.calls.count()).toBe(1);
```

Comment:
Let's make this assertion first. If it's 0, the previous line would throw with a weird error.

Reply:
Done